### PR TITLE
Added python generic example project

### PR DESCRIPTION
**Pull request by ankit-dq**

### CODE REVIEW

File: Python/python_generic/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+![logo](./docs/logo-python-generic.png)
+
+# axe DevTools Python API Example
+
+Using axe DevTools Python, you can integrate axe DevTools into your existing testing environment. This example project demonstrates how axe DevTools Python is used to detect accessibility issues and generates reports in CSV, HTML, JSON, and XML formats.
+
+## Prerequisites
+
+Python 3 (does not support python 2) along with the virtual environment (venv or pyenv)
```
Comment (jthickens) [inline Python/python_generic/README.md:9]:
```suggestion
Python 3 (does not support python 2) along with the virtual environment (venv or pyenv).
```

File: Python/python_generic/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+2. Open the project in your favourite python IDE.
+3. Navigate from the root of the repo to this example with the following command:
+
+```sh
+cd Python/python_generic
+```
+
+## Binary Reporter
+
+Download the respective [Binary reporter](https://docs.deque.com/devtools-html/4.0.0/en/downloads#binary-reporter) and place it in the **_resources_** directory.
```
Comment (jthickens) [inline Python/python_generic/README.md:24]:
```suggestion
Download the respective [Binary reporter](https://docs.deque.com/devtools-html/4.0.0/en/downloads#binary-reporter) for your operating system and place it in the **_resources_** directory.
```

File: Python/python_generic/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+The following command installs all the required dependencies to run this example project.
+
+```sh
+pip install -r requirements.txt
+```
+
+## Run Tests
+
+The **_tests_** directory contains the example test file **`test_example.py`** that analyzes the page `https://broken-workshop.dequelabs.com/` for accessibility issues.
+
```
Comment (jthickens) [inline Python/python_generic/README.md:42]:
```suggestion
```

File: Python/python_generic/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+The following command runs the test file in the **_tests_** directory.
+
+```sh
+python -m test_example
+```
+
+## Test Results
+
+The tests generate results in the **_a11y-results_** directory.
```
Comment (jthickens) [inline Python/python_generic/README.md:52]:
```suggestion
The tests generate results in the **_a11y-results_** directory if you have installed the reporter binary. If you have not completed this step, only JSON reports will be created.
```

**Review by jthickens — DISMISSED** [review]

File: Python/python_generic/README.md (hunk 1 above)
Comment (jthickens) [inline Python/python_generic/README.md:9]:
```suggestion
Python 3 (does not support Python 2) along with the virtual environment (venv or pyenv)
```

File: Python/python_generic/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+## Clone Project
+
+Follow these steps to clone and navigate to the directory:
+1. Clone this repo from GitHub.
+2. Open the project in your favourite python IDE.
+3. Navigate from the root of the repo to this example with the following command:
+
+```sh
+cd Python/python_generic
```
[the remark this answers is not in the document]
Comment (jthickens) [reply Python/python_generic/README.md:19]: ^^

**Review by jthickens — CHANGES_REQUESTED** [review]

File: Python/python_generic/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ankit-dq) [reply Python/python_generic/README.md]: Updated the dir name to python_selenium

**Review by jthickens — APPROVED** [review]
@AdnoC should review the python test files, but everything else lgtm

**Review by AdnoC — APPROVED** [review]

**Pull request merged**